code/index.html

```diff
@@ -1,30 +1,89 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css">
+  <link rel="preload" as="image" href="./doc-image-small.png">
+  <title>Book A Doc | Doctors Online</title>
+</head>
+
+<body>
+
+  <!--- Hero -->
+  <header>
+    <div class="bg-img">
+
+      <!-- Hero Text -->
+      <div class="hero-text">
+        <i class="logo fas fa-user-md" aria-hidden="true"></i>
+        <h1>Book a Doc</h1>
+        <p>Find a doctor or get medical advice</p>
+        <button> <a class="hero-btn" href="#book">Book Now</a>
+        </button>
+
       </div>
+    </div>
+  </header>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+  <!-- Section -->
+  <section class="section-container">
+    <a class="anchor" name="book"></a>
```

Comment:
Hmm, I don't think you need `name` attribute in anchor - `name` attribute is reserved for form inputs only